### DISCUSSION TITLE
"hide" not working 100% on Ubuntu / WSL

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-lualine/lualine.nvim#1295] [General]
On my Mac it works perfectly fine when I trigger the hide function. However on Ubuntu / WSL when I hide lualine, I still see the file name, like this:

<img width="678" alt="image" src="https://github.com/user-attachments/assets/5c9e4129-361b-48d4-a58a-a22bf28dd10a">

Lualine actually hides but I still see the file name (gcp_dump_ago.json).

Ubuntu / WSL requires any extra step?

The command I'm using:
`require('lualine').hide()`

**Comment by iton0** [comment]
is it maybe using ur default statusline? you could try changing your default stuatusline to a space `vim.o.statusline = ' '`. If that does not work, after you hide lualine type `echo &statusline` into your command line and what is the output?

**Reply by rodhash** [reply]
> is it maybe using ur default statusline? you could try changing your default stuatusline to a space `vim.o.statusline = ' '`. If that does not work, after you hide lualine type `echo &statusline` into your command line and what is the output?

sorry delay my notification pop up was a mess.

Today I got the same issue on my Mac running Nvim `v0.11.0-dev-1028+g6dad1f9f1` and setting the default statusline to a space worked!

Thank you.
